lessons/Algebra2/exploring-linear-models/langs/en-us/pages/other-forms-linear-models.adoc

```diff
@@ -0,0 +1,77 @@
+= Other Forms of Linear Models
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Evaluate @show{(code '(lr-plot states-table "state" "college-or-higher" "median-income"))}. What is @math{R^2}? @fitb{5em}{@ifsoln{}}
```

Comment:
this page references `median-income`
the title for that column in the starter file is `median-household-income` 
one of them will need to change.

Reply:
Good catch! Fixed.